ilc/client/WrapApp.spec.js

```diff
@@ -0,0 +1,291 @@
+import WrapApp from './WrapApp';
+import chai from 'chai';
+import sinon from 'sinon';
+
+describe('WrapApp', () => {
```

Comment:
1. You're missing test cases for unmount (when we have wrapper or app rendered).
2. I would try to write utility function to check state of all callbacks at once. Ex. `expectBallbacksToBe(callbacks, 'mounted')`
3. Separately we need to test the fact that App Wrapper always receives `getCurrentBasePath() = '/'`, unique appId and it's own props while wrapped app receives "own" props from registry.
4. We also need to check that class also works with callbacks provided as array of functions. See `#canonizeCallbacks`